charts/dvpe-deployment-gloo/README.md

```diff
@@ -236,28 +239,31 @@ The following table lists the configurable parameters of the chart and its defau
 | gloo.authConfig.spec.configs.clientCredentialsPlugin.config.oidcUrl | string | `nil` | `oidcUrl`  - where the access token can be verified at the IDP |
 | gloo.authConfig.spec.configs.clientCredentialsPlugin.enabled | bool | `false` | If `enabled` set to true the machine to machine plugin will be used |
 | gloo.authConfig.spec.configs.clientCredentialsPlugin.name | string | `"AuthClientCredentials"` | `Name` of the auth client credentials plugin |
+| gloo.authConfig.spec.configs.oauth.cache.cookieName | string | `"auth0-session"` | `CookieName` of the session cookie. Need to be `auth0-session` otherwise the TokenValidation plugin will not validate the cookie |
```

Comment:
you could add a validation rule in values.schema.json that ensures correct naming